components/script/dom/webidls/XRViewerPose.webidl

```diff
@@ -7,4 +7,6 @@
 [SecureContext, Exposed=Window] interface XRViewerPose {
   // readonly attribute XRRigidTransform transform;
   // readonly attribute FrozenArray<XRView> views;
+  // workaround until we have FrozenArray
+  sequence<XRView> views();
```

Comment:
Drop a link to https://github.com/servo/servo/issues/10427#issuecomment-449593626.